gradle/libs.versions.toml

```diff
@@ -56,7 +56,7 @@ timber = "5.0.1"
 chart = "3.1.0"
 leakcanary = "2.14"
 mockk = "1.13.10"
-slf4j = "1.7.36" # don't update to 2.x, unless the next one is also updated
+slf4j = "2.0.13" # don't update to 2.x, unless the next one is also updated
```

Comment:
Don't update this, as it will break logging.

Reply:
Indeed its written there!